modules/tinymce/CHANGELOG.md

```diff
@@ -66,6 +66,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Renamed the `formatselect` toolbar button and `blockformats` menu item to `blocks` #TINY-8328
 - Renamed the `styleselect` toolbar button and `formats` menu item to `styles` #TINY-8328
 - Renamed the `lineheight_formats` option to `line_height_formats` #TINY-8328
+- The `editor.editorCommands.queryCommandSupported` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.editorCommands.queryCommandState` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.editorCommands.queryCommandValue` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.editorCommands.execCommand` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.queryCommandSupported` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.queryCommandState` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.queryCommandValue` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.execCommand` function will no longer fallback to the browsers native command #TINY-7829
+- The `editor.editorCommands.queryCommandSupported` function will now return `false` on removed editors #TINY-7829
+- The `editor.editorCommands.queryCommandState` function will now return `false` on removed editors #TINY-7829
+- The `editor.editorCommands.queryCommandValue` function will now return empty strings on removed editors #TINY-7829
+- The `editor.editorCommands.execCommand` function will now return false on removed editors #TINY-7829
+- The `editor.queryCommandSupported` function will now return `false` on removed editors #TINY-7829
+- The `editor.queryCommandState` function will now return `false` on removed editors #TINY-7829
+- The `editor.queryCommandValue` function will now return empty strings on removed editors #TINY-7829
+- The `editor.execCommand` function will now return false on removed editors #TINY-7829
```

Comment:
This all seems quite verbose. Could we trim it down to just something like this? We can always expand more in the documentation if we need to, but I just feel this is too verbose.
```
- The Editor commands APIs will no longer fallback to executing the browsers native command functionality #TINY-7829
- The Editor query command APIs will now return `false` or an empty string on removed editors #TINY-7829
```

If you don't do this, then `false` in the last line needs backticks around it.